deliverables/README.md

```diff
@@ -9,60 +9,123 @@ To answer questions, please just put the answer on the next line after the quest
 Example:  
 
 **What is your name?**
-Sir Student of Ruby
+Nolawee Mengist
 
 **What is your quest?**
-I seek the Rails  
+Make a stupid amount of money
 
 **What is your favorite color?**
-~~Green~~ edit:  No, blue.  Blue!  
+Red 
 
 For projects on Code Academy, or other external exercises, please just paste a link into your deliverables file where we can see the result.  
 
 ### Section 1: The Front End
 #### Required - Answer these questions in the deliverables file, and complete the following. 
-What is the role of HTML in a web page?  
+What is the role of HTML in a web page? 
+It is the standard markup language used for making webpages. HTML is the backbone of all websites and translates images, video and text into a format that your browser can understand. 
+
 What is the role of CSS in a web page?  
-What is the role of Javascript in a web page?  
+CSS stands for Cascading Style Sheets and is used to adjust the visual asthetics of a webpage.
 
+What is the role of Javascript in a web page?  
+Javascript is used to affect the behavior of HTML pages. It allows you to let the designer dynamically add functionality to their website.
 ---
 
 ### Section 2: HTML and CSS 101
 
 #### meta
 Why do we separate HTML and CSS?  
-
+We seperate HTML and CSS to make maintaining code easier, keeps code more efficent and it's standard programming practice.
```

Comment:
More precisely, because they have different functions.  It's true that it does make maintaining code easier, but it's better to understand *how exactly* it makes it easier.  In this case, by separating functions.  Google "Single Responsibility Principle" for a little philosophy on this.